tests/e2e/koperator_suite_test.go

```diff
@@ -64,6 +64,7 @@ var _ = When("Testing e2e test altogether", Ordered, func() {
 	testInstallKafkaCluster("../../config/samples/simplekafkacluster_ssl.yaml")
 	testProduceConsumeInternalSSL(defaultTLSSecretName)
 	testUninstallKafkaCluster()
+	testWebhooks()
```

Comment:
With the current implementation where webhook prerequisites are decoupled, the logical trouble I saw was that webhook testing would end up doing the install testing for the big CRs before those dedicated tests happen.  
 
I'll give the example we have regarding KafkaTopic :  in order to test anything about KafkaTopic we need a KafkaCluster to be up which means we would end up testing KafkaCluster installation (without meaning to) before the dedicated KafkaCluster installation and functional tests.

I saw that as not good so I dedided to place webhook testing towards the end as a category of its own which can then rely on dedicated install and functional tests working (and having been addressed in their own sections beforehand).

I can be convinced otherwise, especially within the context of this conversation : https://github.com/banzaicloud/koperator/pull/1031#discussion_r1293744826 .